libimage/normalize_test.go

```diff
@@ -132,7 +132,7 @@ func TestNormalizeTaggedDigestedString(t *testing.T) {
 		{"localhost/fedora:anothertag" + digestSuffix, "localhost/fedora" + digestSuffix},
 		{"localhost:5000/fedora:v1.2.3.4.5" + digestSuffix, "localhost:5000/fedora" + digestSuffix},
 	} {
-		res, err := normalizeTaggedDigestedString(test.input)
+		res, _, err := normalizeTaggedDigestedString(test.input)
```

Comment:
This should probably be confirming that the second value isn't `nil`, and matches `res`.